Sources/Rendering/OpenGL/Texture/index.d.ts

```diff
@@ -244,7 +244,8 @@ export interface vtkOpenGLTexture extends vtkViewNode {
    * @param numComps The number of components in the texture.
    * @param dataType The data type of the texture.
    * @param data The raw data for the texture.
-   * @param preferSizeOverAccuracy Whether to prefer texture size over accuracy.
+   * @param [preferSizeOverAccuracy=false] Whether to prefer texture size over accuracy. Defaults to false.
+   * @param [ranges] The ranges of the data (optional).
```

Comment:
The precomputed ranges of the data (optional). Provided to prevent computation of the data ranges.

Reply:
Updated.